client/src/components/CFPIntro.js

```diff
@@ -17,9 +17,7 @@ const CFPIntro = () => (
       <li><a href="https://summit2017.reversim.com">Reversim Summit 2017</a></li>
     </ul>
     <p>We are interested in many topics, everything including: Software development methodologies, Product management for developers, UX, Startups, Mobile, Web, Devops, Data processing, Scaling, Software company culture, Tooling, etc...</p>
-    <p>We want deep, technical sessions. We are not looking for “intro to something software” or “something software 101”. We’re looking for something of greater depth. An interesting project you worked on, research, etc. Not entry level.</p>
-    <p>However, we are open to session “intro to something that isn’t software”, as long is this something is of general interest, for example “intro to moonwalking and breakdance”.</p>
-    <p>We are looking for deep, hard-core technical sessions which exemplify outstanding and unique work or learnings which derive from your personal experience.</p>
+    <p>We want deep, technical sessions, yet ones that can be beneficial to a general audience of  your assigned track. From our experience, talks that tells a story, preferably yours, work best. We want stories which exemplify outstanding and unique work or learning which derive from your personal experience. Any technical improvement you achieved by non trivial means, that might help others improve, is worth sharing.</p>
```

Comment:
I prefer using `sessions` and `session` rather than `talks` and `talk` across the board in a consistent way, so if you don't mind please replace `talks` with `sessions` in this sentence.

Reply:
fixed